testing/forge-script/forge-config.yaml

```diff
@@ -24,10 +24,10 @@ deployment:
   contracts_path: "contracts"  # give the path till contracts, if the repository is the contract folder itself, then leave it empty
   script_path: "script/DeployAndCallERC20.s.sol"  # this must be relative to the repository path + contracts_path(if applicable)
   contract_name: "DeployAndCallERC20"
+  dependency:
+    type: "none"  # type can be `git` or `local` or `none`
+    path: "script/berps/dependency/dependency.sh"
   rpc_url: "http://HOST_IP_ADDRESS:8547"  # If you spin up local devnet via kurtosis, then public port is 8547
   wallet:
     type: "private_key"  # currently only private_key wallet is supported. Do not change the type.
-    value: "0xfffdbb37105441e14b0ee6330d855d8504ff39e705c3afa8f859ac9865f99306"
-  dependency:
-    status: false
-    path: "dependency.sh"
+    value: "0xYOUR_PRIVATE_KEY"
```

Comment:
leave as the 0xff and then add a comment

Reply:
Addressed.